Resources/template.json

```diff
@@ -141,46 +141,72 @@
-        }
+      "type": "string",
+      "defaultValue": "10.0",
+      "metadata": {
+        "description": "First 2 octects of VNet Private IP address range (VNet prefix)"
+      }
     },
     "virtualNetworkRoutedSubnets": {
-        "type": "array",
-        "metadata": {
-            "description": "Array of subnets to create"
-        }
+      "type": "array",
+      "metadata": {
+        "description": "Array of subnets to create"
+      }
     },
     "virtualNetworkNonRoutedSubnets": {
-       "type": "array",
-       "metadata": {
-           "description": "Array of subnets to create"
-     }
-    },
-      "msqlSubnetName": {
-        "type": "string",
-        "metadata": {
-          "description": "Used for managed sql deployment"
-        }
-      },
-      "deployManageSql": {
-        "type": "bool",
-        "metadata": {
-          "description": ""
-        }
-      },
-      "vnetResourceGroup": {
-        "type": "string",
-        "metadata": {
-          "description": "Name of the resource group vNets are deployed"
-        }
+      "type": "array",
+      "metadata": {
+        "description": "Array of subnets to create"
+      }
+    },
+    "msqlSubnetName": {
+      "type": "string",
+      "metadata": {
+        "description": "Used for managed sql deployment"
       }
+    },
+    "deployManageSql": {
+      "type": "bool",
+      "metadata": {
+        "description": ""
+      }
+    },
+    "vnetResourceGroup": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the resource group vNets are deployed"
+      }
+    },
+    "appGatewayNetworkSecurityGroup": {
+      "type": "string",
+      "metadata": {
+        "description": "The network security group assigned to the ASEs subnet.  The rules on this group will be altered by the release so this group should not be shared with other subnets."
+      }
+    },
+    "appGatewayVnetName": {
+      "type": "string",
+      "defaultValue": "",
+      "metadata": {
+        "description": "Name the VNet the ASE subnet is part of. Only needed if envType is Prod."
+      }
+    },
+    "appGatewaySubnetName": {
+      "type": "string",
+      "metadata": {
+        "description": "Subnet gateway name"
+      }
+    },
+    "appGatewayIncomingIpAddressRange": {
+      "type": "string",
+      "defaultValue": "0.0.0.0/1,128.0.0.0/2,192.0.0.0/3,224.0.0.0/4",
+      "metadata": {
+        "description": "Restricts HTTP(S) traffic to App Gateway to the passed in range"
+      }
+    }
   },
   "variables": {
     "dfcDevOpsTemplatesUrl": "https://raw.githubusercontent.com/SkillsFundingAgency/dfc-devops/master/ArmTemplates/",
-    "localTemplatesUrl": "https://raw.githubusercontent.com/SkillsFundingAgency/dfc-shared/master/Resources/",
+    "localTemplatesUrl": "https://raw.githubusercontent.com/SkillsFundingAgency/dfc-shared/NCSD-986_appgatewaynsg/Resources/",

```

Comment:
still referring to the branch rather than master

Reply:
Doh, fixed